lib/mix/lib/mix/tasks/test.ex

```diff
@@ -122,8 +122,8 @@ defmodule Mix.Tasks.Test do
   ## Configuration
 
     * `:test_paths` - list of paths containing test files, defaults to
-      `["test"]`. It is expected all test paths to contain a `test_helper.exs`
-      file.
+      `["test"]` if the `test` path exists, otherwise, it defaults to `[]`. It
+      is expected all test paths to contain a `test_helper.exs` file.
```

Comment:
We can rephrase this as:

> [...] defaults to `["test"]` if the `test` directory exists, otherwise it defaults to `[]`. It is expected that all test paths contain a `test_helper.exs` file.

Wdyt?